appscale/tools/appscale_tools.py

```diff
@@ -62,11 +66,39 @@ class AppScaleTools(object):
   # A regular expression that matches files compressed in the zip format.
   ZIP_REGEX = re.compile(r'.zip\Z')
 
+
   # A str that contains all of the authorizations that an AppScale cloud
   # administrator should be granted.
   ADMIN_CAPABILITIES = "upload_app"
 
 
+  # AppScale repository location on local filesystem.
+  APPSCALE_REPO = "~/appscale"
+
+
+  # Command to run the Bootstrap.
+  RUN_BOOTSTRAP_COMMAND = "bash bootstrap_force_upgrade.sh --local-ip"
+
+
+  # String of commands to cd into the AppScale repository and run the Bootstrap.
+  BOOTSTRAP_COMMAND = "cd " + APPSCALE_REPO + ";" + RUN_BOOTSTRAP_COMMAND
+
+
+  # Command to run the upgrade script from /appscale/scripts directory.
+  UPGRADE_SCRIPT = "python " + APPSCALE_REPO + "/scripts/upgrade.py"
+
+
+  # Git command to get the latest tags for a repository.
+  GIT_COMMAND = "git ls-remote --tags https://github.com/AppScale/appscale.git"
+
+
+  # Location of the upgrade status file on the remote machine.
+  UPGRADE_STATUS_FILE_LOC = '/var/log/appscale/upgrade-status-'
+
+  # .JSON file extention
+  JSON_FILE_EXTENTION = ".json"
```

Comment:
This doesn't make sense to me to have as a constant. If you anticipate needing to change the extension for the status file in the future, you could define it here, but you'd probably want to call it something more descriptive.